test/message/context.spec.ts

```diff
@@ -0,0 +1,171 @@
+import { expect } from 'chai';
+import { verifyMessage, signMessage, generateKey, readSignature, readMessage, decryptMessage, encryptMessage, readKey } from '../../lib';
+import { VERIFICATION_STATUS } from '../../lib/constants';
+
+// verification without passing context should fail
+// verification passing wrong context should fail
+// verification of unsigned notation data should fail
```

Comment:
Could you also add a test case of verifying a signature that doesn't have any context notation data, to check that it fails to verify (when a context is passed to `verifyMessage`)? I know it's almost the same as the unsigned one, but since it's the main case that we care about, I think it would be nice to have an explicit test for it (and not just one "hidden" as an edge case)